lib/bundler/injector.rb

```diff
@@ -101,5 +119,147 @@ def append_to(gemfile_path, new_gem_lines)
         f.puts new_gem_lines
       end
     end
+
+    # evalutes a gemfile to remove the specified gem
+    # from it.
+    def remove_deps(gemfile_path)
+      # get initial snap shot of the gemfile
```

Comment:
I find that comments like these don't really offer any useful information as they're only describing what the code is doing, which we can see already from the code. There is also the risk that they could fall out of date and end up being wrong the in the future.

I would prefer we remove these